superset/translations/babel.cfg

```diff
@@ -19,6 +19,7 @@
 [jinja2: superset/**/templates/**.html]
 [javascript: superset-frontend/src/**.js]
 [javascript: superset-frontend/src/**.jsx]
+[javascript: superset-frontend/src/**.ts]
```

Comment:
@zhaoyongjie let's make sure translations are properly handled after monorepo lands!

Reply:
@villebro I have rebase master on [stage-2](https://github.com/apache/superset/pull/17552) PR

Reply:
I will add plugins and packages to `babel.cfg`

Reply:
Cool thanks @zhaoyongjie !